packages/patternfly-4/react-core/src/components/ClipboardCopy/ClipboardCopyExpanded.tsx

```diff
@@ -30,16 +32,19 @@ export class ClipboardCopyExpanded extends React.Component<ClipboardCopyExpanded
   }
 
   render() {
-    const { className, children, onChange, isReadOnly, ...props } = this.props;
-    return (
-      <div
-        suppressContentEditableWarning
-        ref={this.contentRef}
-        className={css(styles.clipboardCopyExpandableContent, className)}
-        onInput={(e: any) => onChange(e.target.innerText, e)}
-        contentEditable={!isReadOnly}
-        {...props}
-      />
-    );
+    const { className, children, onChange, isReadOnly, isCode, ...props } = this.props;
+    const clipboardCopyExpandableComponent =
+    <div
+      suppressContentEditableWarning
+      ref={this.contentRef}
+      className={css(styles.clipboardCopyExpandableContent, className)}
+      onInput={(e: any) => onChange(e.target.innerText, e)}
+      contentEditable={!isReadOnly}
+      {...props}
+    />;
+
+    return (isCode ?
```

Comment:
could probably work a little cleaner like this:
```
return isCode ? <pre>{clipboardCopyExpandableComponent}</pre> : clipboardCopyExpandableComponent
```

Reply:
Fixed -- thank you Joachim!